index.html

```diff
@@ -0,0 +1,335 @@
+                        <div class="card-text-main ">Back End Dev</div>
+                        <div class="work-card-dot-em2"></div>
+                        <div class="work-card-dot-list">
+                            <div class="card-icon ">
+                            </div>
+                        </div>
+                        <div class="work-card-dot-em"></div>
+                        <div class="card-text-main ">2015</div>
+                    </div>
+                    <h2 class="card-subtext ">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h2>
+
+                    <ul class="grid">
+                        <li class="item">html</li>
+                        <li class="item">css</li>
+                        <li class="item">javascript</li>
+                    </ul>
+                    <div class="card-button ">
+                        <button type="button" class="button1">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    
+        
+
+          <div class="work-card layout">
+            <div class="Work-card-head ">
+                <div class="Work-card-body">
+                    <div class="Work-card-img4 ">
+                    </div>
+                    <h3 class="work-card-title ">Multi-Post<br>Stories</h3>
+                    <div class="work-card-dot ">
+                        <div class="card-text-main">CANOPY</div>
+                        <div class="work-card-dot-s"></div>
+                        <div class="work-card-dot-list">
+                            <div class="card-icon ">
+                            </div>
+                        </div>
+                        <div class="work-card-dot-em"></div>
+                        <div class="card-text-main ">Back End Dev</div>
+                        <div class="work-card-dot-em2"></div>
+                        <div class="work-card-dot-list">
+                            <div class="card-icon ">
+                            </div>
+                        </div>
+                        <div class="work-card-dot-em"></div>
+                        <div class="card-text-main ">2015</div>
+                    </div>
+                    <h2 class="card-subtext ">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h2>
+
+                    <ul class="grid">
+                        <li class="item">html</li>
+                        <li class="item">css</li>
+                        <li class="item">javascript</li>
+                    </ul>
+                    <div class="card-button ">
+                        <button type="button" class="button1">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <section class="About ">
```

Comment:
Your design doesn't quite follow the template design.

Here's your design:

<img width="246" alt="Screen Shot 2022-05-27 at 17 09 11" src="https://user-images.githubusercontent.com/82426682/170795172-bbbe7f90-ca25-480c-83d2-555810b0da5e.png">

Here's the template design:
<img width="216" alt="Screen Shot 2022-05-27 at 17 10 20" src="https://user-images.githubusercontent.com/82426682/170795267-71b6922f-2b35-48ac-89d4-e7024a97522e.png">

Do you see how different the overall layout (margins, padding, etc.) and the `font-family` and `font-size` are? Kindly fix this.